src/leaseStatisticReport/components/LeaseInvoicingConfirmationReport.js

```diff
@@ -81,66 +84,31 @@ class LeaseInvoicingConfirmationReport extends PureComponent<Props, State> {
-            : <FormText className='no-margin' style={{whiteSpace: 'nowrap'}}>-</FormText>,
-        });
-      } else if (field.key === LeaseInvoicingReportPaths.AREA) {
-        columns.push({
-          key: field.key,
-          text: field.label,
-          sortable: false,
-          renderer: (area) => area
-            ? <FormText className='no-margin' style={{whiteSpace: 'nowrap'}}>{`${formatNumber(area)} m²`}</FormText> 
-            : <FormText className='no-margin' style={{whiteSpace: 'nowrap'}}>-</FormText>,
-        });
-      } else if(field.key === LeaseInvoicingReportPaths.LEASE_ID){
-        columns.push({
-          key: field.key,
-          text: field.label,
-          renderer: (identifier) => identifier
-            ? <FormText className='no-margin' style={{whiteSpace: 'nowrap'}}>{identifier}</FormText> 
-            : <FormText className='no-margin' style={{whiteSpace: 'nowrap'}}>-</FormText>,
-        });
-      } else if(field.key === LeaseInvoicingReportPaths.STATE){
-        columns.push({
-          key: field.key,
-          text: field.label,
-          renderer: (state) => state
-            ? <FormText className='no-margin' style={{whiteSpace: 'nowrap'}}>{getInvoiceState(state)}</FormText> 
-            : <FormText className='no-margin' style={{whiteSpace: 'nowrap'}}>-</FormText>,
-        });
-      } else if(field.key === LeaseInvoicingReportPaths.TOTAL_AMOUNT ||
-        field.key === LeaseInvoicingReportPaths.BILLED_AMOUNT ||
-        field.key === LeaseInvoicingReportPaths.OUTSTANDING_AMOUNT ||
-        field.key === LeaseInvoicingReportPaths.RENT ||
-        field.key === LeaseInvoicingReportPaths.PAID_AMOUNT) {
-        columns.push({
-          key: field.key,
-          text: field.label,
-          sortable: false,
-          renderer: (amount) => amount
-            ? <FormText className='no-margin' style={{whiteSpace: 'nowrap'}}>{`${formatNumber(amount)} €`}</FormText> 
-            : <FormText className='no-margin' style={{whiteSpace: 'nowrap'}}>-</FormText>,
-        });
-      } else {
-        columns.push({
-          key: field.key,
-          text: field.label,
-          sortable: false,
-        });
-      }
+      columns.push({
+        key: field.key,
+        text: field.label,
+        renderer: (value) => {
+          let isBold = false;
+          let outputValue = value || '-';
+
+          if (field.choices && value) {
+            outputValue = getDisplayName(field.choices, value);
+          } else if (field.format && value) {
+            outputValue = getFormattedValue(field.format, value);
+            isBold = field.format === LeaseStatisticReportFormatOptions.BOLD || field.format === LeaseStatisticReportFormatOptions.BOLD_MONEY;
+          }
+
+          return (
+            <FormText className='no-margin' style={{ whiteSpace: 'nowrap' }}>
+              {isBold ? <b>{outputValue}</b> : outputValue}
```

Comment:
`<strong>` is generally preferred for semantic reasons over `<b>` these days, I believe, but it works either way.

Reply:
I'll change this to `<strong>`